salt/onboarding.sls

```diff
@@ -10,16 +10,14 @@ update all packages:
     - refresh: True
     - dist_upgrade: True
 
-{%- if 'fqdn_ip4' in grains %}
-{{ grains.fqdn_ip4[0] }}:
+{{ pillar.main_ipv4 }}:
```

Comment:
This means adding IPs for all servers in Pillar?

Reply:
I have changed how this works, refer to new server onboarding and netplan docs.